controllers/oauth-server/index.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+"use strict";
+
+const express = require('express');
+const oauthClientsModel = require('../../models/oauth-clients');
+const router = express.Router(); 
+const ONE_MINUTE = 1000 * 60;
+const ONE_HOUR = ONE_MINUTE * 60;
+
+function encrypt(input, secret) {
+  // TODO Implement preper encryption
+  return input;
+}
+
+function generateAuthCode(oauthClient, user, scope) {
+  const authCode = {
+    type: "code",
+    id: user.username,
+    scope: scope,
+    expires_at: new Date() + ONE_MINUTE
```

Comment:
Should this be Date.now() + ONE_MINUTE? new Date() + ONE_MINUTE gives me a concatenated string on my machine.